cluster/gce/gci/configure-helper.sh

```diff
@@ -752,10 +752,10 @@ function start-kube-apiserver {
   fi
   local -r src_dir="${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty"
 
-  if [[ -n "${KUBE_USER:-}" ]]; then
+  if [[ -n "${KUBE_USER}" ]]; then
```

Comment:
KUBE_USER is now initialized early on in the setup (on line 1119) if its not set. So its redundant to do it here.